base/support_menus/CharsMenu.ts

```diff
@@ -363,8 +360,8 @@ export class CharsMenu {
 
         this.set_arrows(false, false);
 
-        this.window.close(this.close_callback, false);
-        this.close_callback = null;
+        this.window.close(callback, false);
+        callback = null;
```

Comment:
This line seems to be unnecessary and maybe dangerous if you're going to use this callback on the caller side without resetting it...